src/main/resources/static/inputs.html

```diff
@@ -6,6 +6,13 @@
             function handlePress(element) {
                 document.querySelector("#output").innerText = element.value
             }
+
+            function addShadowDom(element) {
```

Comment:
Please move to separate page, there is no reason to have the mixed page for inputs and shadow-dom

Reply:
cc @valfirst

Reply:
makes sense